checks/runtimeCheck.go

```diff
@@ -23,12 +24,17 @@ func checkAndReturnRuntime() (runtimeConfig, error) {
 }
 
 func latestAgentTag(r *runtimeConfig) error {
-	resp, err := http.Get(r.agentVersionUrl)
+	client := &http.Client{
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+	}
+	resp, err := client.Get(r.agentVersionUrl)
 	if err != nil {
 		return err
 	}
 	defer resp.Body.Close()
-	if resp.StatusCode >= 300 {
+	if resp.StatusCode > 302 {
```

Comment:
Is it that we're handling 300 (multiple choices), 301 (moved permanently) and 302 (found)? Or should this be `statusCode < 300 || statusCode == 302`?

Reply:
Yeah i was thinking that as well actually, I'll add it.